### DISCUSSION TITLE
Shelly 2.5 - Splittare i canali

### DISCUSSION THREAD

**Discussion by naverre** [indomus/forum#2460] [HUB: Home Assistant]
Salve, ho da poco installato uno Shelly 2.5 che controlla una luce in cucina e una nel soggiorno, ma integrando il dispositivo in HA posso assegnarlo ad una sola stanza. C'è un modo per assegnare i due canali ad ambienti diversi? Grazie

**Comment by snakuzzo** [comment]
Si chiamano "aree" e oggettivamente non servono proprio a niente.
Comunque no, non puoi. L'area si può impostare da GUI solo a livello di device

**Reply by naverre** [reply]
Ti ringrazio per la risposta
